experimentation/src/main/java/com/automattic/android/experimentation/ExPlat.kt

```diff
@@ -25,37 +22,42 @@ public class ExPlat internal constructor(
 
     private var anonymousId: String? = null
 
-    init {
-        coroutineScope.launch {
-            refresh()
-        }
-    }
-
-    override fun configure(anonymousId: String?) {
-        clear()
+    override fun initialize(anonymousId: String) {
```

Comment:
💭 

Not for now, and just thinking out loud (and maybe missing more context): perhaps another approach for the API could be to have a builder class (or even a static factory method on the interface?) that will take the `anonymousId` and initialize the classes, so that they wouldn't be able for a client to be create these classes without properly initializing the SDK.

Reply:
Oh I see that's kinda what `VariationsRepository.create()` does? Can't it also take the `anonymousId`? 🤔

Reply:
> Oh I see that's kinda what VariationsRepository.create() does? Can't it also take the anonymousId? 🤔

Not really, because user can change during the session (logout/sign in). `VariationsRepository` is meant to be a singleton, so `VariationsRepository#create` will be called once for the app lifecycle.